misc/life_mngr/s5_trigger.sh

```diff
@@ -0,0 +1,214 @@
+#! /bin/bash
+
+MAX_ARRAY=6
+
+started="started"
+stopped="stopped"
+SOS="sos"
+UOS="uos"
+SHUTDOWN="shutdown"
+ACKED="acked"
+
+#config
+IP_ADDR="127.0.0.1"
+SOCKET_PORT="8193"
+
+#send message by socket
+send_message()
+{
+	message="$1"
+	echo "sending: $message"
+	echo -ne "$message" >&6 &
+}
+
+#read message by socket
+read_message()
+{
+	read -r -d $'\0' ret_msg <&6
+}
+
+#send message by tty
+send_pre_message()
+{
+	message="$1"
+	echo "sending: $message"
+	echo -ne "$message" > /dev/ttyS1
+}
+
+#read message by tty
+read_pre_message()
+{
+	read -r -d $'\0' ret_msg < /dev/ttyS1
+}
+
+power_off_post_vms() {
+	vm_list=$(acrnctl list)
+	echo $vm_list
+
+	array=($vm_list)
+	num=${#array[@]}
+	echo "vm array size:" $num
```

Comment:
```suggestion
	echo "Number of VMs:" $num
```

Would this be appropriate?